samples/peripheral/802154_phy_test/prj.conf

```diff
@@ -44,6 +44,7 @@ CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE=y
 # Enable ring buffers
 CONFIG_RING_BUFFER=y
 
-CONFIG_NRF_802154_TEMPERATURE_UPDATE=n
+# Set temperature sensor update period in ms
+CONFIG_NRF_802154_TEMPERATURE_UPDATE_PERIOD=10000
```

Comment:
Is there a need to update default configuration value?

Reply:
The default value (1 minute) can in my opinion be too high, if the user would like to check if the sensor and associated command works properly, e.g. by simply putting a finger on the SoC, they won't observe any change for a minute (worst case scenario). Additionally, redefining the value in proj.conf makes it easier for the user to find the config and adjust it as per their needs. However, if such approach is not the right one, this can be removed without any negative impact on the sample.

Reply:
I understand your concern. I believe we should not focus on engineers putting fingers on SoCs, but rather on RF tests in environmental chambers that can verify RF performance in broad range of temperatures. I don't know what are typical temperature change rates in such chambers. Is 10 second period enough, or too long for such tests? Would it be OK to measure temperature much more often? The lower intervals, the quicker test setups we can support.

Reply:
Considering that such chamber needs to not only change and stabilize the internal air temperature, but the temperature of the SoC itself as well, I doubt that specific heat capacity of the SoC and the PCB would allow for quicker times of changing and achieving stable temperature. Nevertheless, I briefly tested the sample with temperature update periods as low as 100ms, and the tool seemed to work in a stable and correct manner.